src/WebJobs.Extensions.DurableTask/Options/AzureStorageOptions.cs

```diff
@@ -172,6 +172,13 @@ public string TrackingStoreConnectionStringName
         /// <value>A boolean indicating whether we use the legacy partition strategy. Defaults to false.</value>
         public bool UseLegacyPartitionManagement { get; set; } = false;
 
+        /// <summary>
+        /// Determines whether to use the table partition management strategy.
+        /// This strategy reduces expected costs for an Azure Storage V2 account.
+        /// </summary>
+        /// /// <value>A boolean indicating whether to use the table partition strategy. Defaults to false.</value>
```

Comment:
there seems to be an extra set of `///` here :)
```suggestion
        /// <value>A boolean indicating whether to use the table partition strategy. Defaults to false.</value>
```